clients/fossid-webapp/src/test/kotlin/FossIdClientNewProjectTest.kt

```diff
@@ -107,8 +101,10 @@ class FossIdClientNewProjectTest : StringSpec({
 
     "Scans for project can be listed when there is no scan" {
         service.listScansForProject("", "", PROJECT_CODE) shouldNotBeNull {
-            checkResponse("list scans")
-            toList<Scan>() should beEmpty()
+            checkResponse("list scans", true)
```

Comment:
`true` is the default for the `withDataCheck` parameter and should be omitted here and in several other places.